### ISSUE TITLE
Announcements

### ISSUE THREAD

**Issue by EdricChan03** [Chan4077/chan4077.github.io#38]
This issue will contain any new announcements about the site.

# TL;DR: Summarised list of announcements
- [**15 Dec 2018**: Redesign of website](https://github.com/Chan4077/chan4077.github.io/issues/38#issuecomment-447541721)
- [**15 Jan 2019**: GitHub Actions testing](https://github.com/Chan4077/chan4077.github.io/issues/38#issuecomment-454434175)
- [**26 Feb 2019**: TIL posts](https://github.com/Chan4077/chan4077.github.io/issues/38#issuecomment-467469542)
- [**21 Dec 2019**: Note on GitHub Actions](https://github.com/EdricChan03/edricchan03.github.io/issues/38#issuecomment-568191430)
- [**16 Apr 2023**: On Jekyll and SSG](https://github.com/EdricChan03/edricchan03.github.io/issues/38#issuecomment-1509918460)
---
_Last edited on 16 Apr 2023 at 18:14:50 UTC_

**Comment by Chan4077** [comment]
# 15 January 2019

I'm currently testing the new GitHub Actions. Let's see if it works. Check back later!

**Comment by Chan4077** [comment]
# 26 February 2019

I'm adding a new category to the blog: TIL (**T**oday **I** **L**earned) posts

**Comment by EdricChan03** [comment]
# 21 December 2019

I've forgotten to note that the website is now being built with [GitHub Actions](https://github.com/features/actions). Check out the [GitHub Actions workflow](https://github.com/EdricChan03/edricchan03.github.io/blob/website-v2/.github/workflows/site-ci.yml) used to deploy the site!

**Comment by EdricChan03** [comment]
# 16 April 2023

I've been pondering over using a better alternative of Jekyll for a while now, but haven't had the time to put down my thoughts until now: #126
